torch/onnx/utils.py

```diff
@@ -573,7 +587,11 @@ def _model_to_graph(model, args, verbose=False,
             output_wrapped = torch_out  # type: ignore[assignment]
 
         output_tensors, out_desc = torch._C._jit_flatten(tuple(output_wrapped))
-        torch._C._jit_pass_onnx_assign_output_shape(graph, output_tensors, out_desc, _onnx_shape_inference)
+        # NOTE: assign_output_shape pass is not compatible with quantized outputs.
```

Comment:
Nit: Remove "NOTE:". The comment will be shorter and have the same informational content without it.